src/protocol/templates/ping1d.h.in

```diff
@@ -0,0 +1,91 @@
+/*
+* ping1d.h
+* A device API for the Blue Robotics Ping1D echosounder
+*
+*~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+* THIS IS AN AUTOGENERATED FILE
+* DO NOT EDIT
+*~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+*
+*/
+
+#pragma once
+
+#include "Arduino.h"
+#include "Stream.h"
```

Comment:
Where this came from ?

Reply:
they are arduino dependencies. I do not understand the question.